fastlane/testflight.md

```diff
@@ -174,15 +177,34 @@ _Referring to the table below, tap on each **IDENTIFIER** that has a different *
     * Trio WatchKit Extension
 1. Click on the **IDENTIFIER** row.
 1. Scroll down to the "App Groups" capabilies row, click on the "Configure" (or "Edit") button.
-1. Select (or verify the selection for) the "Loop App Group" _(yes, "Loop App Group" is correct)_
+1. Select the "Trio App Group" _(yes, "Trio App Group" is correct)_
 1. Click "Continue".
 1. Click "Save".
 1. Click "Confirm".
 1. Remember to do this for each of three identifiers listed under step 2.
 
 There is an additional identifier, but it does not need the App Group added to it:
+
 * Trio LiveActivity
 
+There is one more identifier: LiveActivity, but it does not require the Trio App Group be added.
```

Comment:
This entire added section +190 through + 207 is redundant and confusing, please remove it. The modfication suggestion is found in-line under App Group and Identifiers.

Reply:
Removed.